webui/react/src/pages/ExperimentList.module.scss

```diff
@@ -2,6 +2,13 @@
   display: flex;
   flex-direction: column;
 
+  tr a[class*='Link_'] {
+    color: inherit;
+
+    &:hover {
+      color: inherit;
+    }
+  }
```

Comment:
**suggestion:** I think when the row is no longer clickable, keeping the links blue will be more important and obvious to the user that they can click on them. Will be easier to not having to add these for every table also.

Reply:
should the name in experiment list be also blue this way?

Reply:
yeah anything that's a link let's make them blue so it's crystal clear what they are. So on the experiment list table, it would be the ID and the Name. I was thinking originally just leaving the ID as the clickable link, but having the name also clickable (when available) really seems to help a lot because often it's the larger target.

Reply:
or maybe underline?

![2021-02-22_10-50-27](https://user-images.githubusercontent.com/12127420/108755225-d5f2bb00-74fb-11eb-8c1f-b6f9cd9051c9.png)
![2021-02-22_10-49-36](https://user-images.githubusercontent.com/12127420/108755227-d68b5180-74fb-11eb-8f46-1e91804d383e.png)